### ISSUE TITLE
gdb debugger not working with latest StateMod

### ISSUE THREAD

**Issue by smalers** [OpenCDSS/cdss-app-statemod-fortran#42]
I tried to use the `gdb` debugger on StateMod 16.00.48 to troubleshoot the runtime issue.  It would not work properly, as if the command line arguments are not being seen.

**Comment by smalers** [comment]
The problem was that the command line was 127 characters and the absolute path folder being seen internally in the software was long and getting truncated.  Although Windows has a limit of 8092 characters for command line and Linux is also similarly long, I set the array size to 512 characters for the full command line in `parse_gfortran.for`.  I'm not going to change `parse.for` because I can't test on Lahey.  Individual files were already set at 256 characters.  This should accommodate full paths used in automated testing.  After fixing, `gdb` was able to run.  However, it was no help in this case because there was not a crash.  The error is handled by the software checks.   I'll leave this issue open until others have run the latest version.

**Comment by Rayrbennett** [comment]
Please, please, please put executable in a simple directory like C/cdss/bin when successfully compiled.  

The long directory paths you’ve set up are ok for code mgt but not for using code for production.  

Ray

**Comment by smalers** [comment]
I previously provided email indicating that a `bat` file could be used to do the copy and other tasks, which can be run in a windows shell.  I also explained in that email how to add aliases to Linux `.bashrc` file, in order to `cd` to the development folder.  A similar alias can be defined to copy the executable.

To get over this hurdle... I added to the `makefile` so that the executable is copied to `/C/cdss/bin`.   I emailed this to Ray.
```
